chore: move EmbedGH live sample into the page (#40426)
mdn/content: files/en-us/web/css/_colon_read-write/index.md

## Examples

### Confirming form details using read-only controls

> You can use `readonly` form controls when you want a user to verify information they have entered earlier, and submit it along with new data in read-write controls.

Reviewer: ```suggestion
You can use [`readonly`](/en-US/docs/Web/HTML/Reference/Attributes/readonly) form controls when you want a user to verify information they entered earlier, which you want to submit with new data in read-write controls.
```

> In the example below, the `:read-only` pseudo-class is used to make the `<textarea>` (a user's address) look like a regular paragraph and the `:read-write` pseudo-class provides a way to highlight the editable `<textarea>` (the delivery instructions):

Reviewer: ```suggestion
In the example below, the {{cssxref(":read-only")}} pseudo-class is used to make the {{htmlelement("textarea")}} (a user's address) look like a regular paragraph. The `:read-write` pseudo-class provides a way to highlight the editable `<textarea>` (the delivery instructions):
```
